pom.xml

```diff
@@ -164,7 +165,7 @@
             </goals>
             <configuration>
               <includes>
-                <include>org/bytedeco/javacpp/presets/*.java</include>
+                <include>org/bytedeco/javacpp/**/*_presets.java</include>
```

Comment:
I think we can simplify this by just not compiling what's in `${project.basedir}/src/gen/java`. Can you try that?

Reply:
This prevent for compiling helper classes too (that may depend on generated classes that will be generated later in the build process)

Reply:
The helper classes are not part of the generated classes. It's not very important, but if the generated classes contain syntax errors, this prevents the build from passing until we erase them manually, so we shouldn't try to compile them at this point.

Reply:
We don't. At this point `compileSourceRoot` is not set, so defaults to `src/main/java` and the `includes` limits the source file to the presets file. Thus neither the generated classes nor the helper classes are compiled. I don't understand what's wrong in this configuration.

Reply:
Ah, yes, it's not part of the same execution. I got mixed up. It looks alright.